packages/coil-oauth-scripts/src/Stream.ts

```diff
@@ -226,22 +241,19 @@ export class Stream {
         const onStop = onPluginDisconnect
 
         const onUpdateAmountInterval = async () => {
-          const sendMaxAmount = await adaptiveBandwidth.getStreamSendMax()
-          if (stream.isOpen()) {
-            stream.setSendMax(sendMaxAmount)
-          }
+          // Round to ignore floating point errors.
+          const newSendMax = Math.round(
+            throughput * 60 * this.currentStreamSendMax()
+          )
+          if (+stream.sendMax < newSendMax) stream.setSendMax(newSendMax)
```

Comment:
We generally prefer Number(string) in the rest of the codebase, but it's not a biggy.

Reply:
Yeah, I prefer `+string` since its more succinct.